R/utils.R

```diff
@@ -52,9 +52,12 @@
 #'
 #' @param old_prior One or more prior distributions in the class `brmsprior`
 #' @param new_prior One or more prior distributions in the class `brmsprior`
+#' @param abort If `TRUE` then if a `new_prior` is provided for which there is
```

Comment:
Some thoughts: 

1. `epidist_prior` could take an argument about how strict to be
2. This "prior_present" could be a mulit-option string arg (i.e "strict", "warn" and "allow-mismatch").
3. I think my preference is a no warning as I can see this being really annoying for users with only very occasionally benefits.

Reply:
Note that users don't call `epidist_prior` it's internal so we'd need a pass through option in `epidist` to allow it to be settable

Reply:
they could call it though right? Its exported and just takes common  arguments?

Reply:
if yes to above we could doc that investigation path or as you say we could pass through the arg.

Reply:
Yep let me investigate this. I did consider if there is some general outer "options" input that we want (like a general `verbose` or `strict` option)

Reply:
I do not think we need something as sophisticated as multistring option here. Proposed solution in comments below (and code).